e2e-tests/contentful/schema.gql

```diff
@@ -1,4 +1,4 @@
-### Type definitions saved at 2021-04-27T13:44:24.013Z ###
+### Type definitions saved at 2021-05-12T15:50:31.482Z ###
```

Comment:
This updates the schema.gql to the current version. This is used for breaking change detection and comparison for #31385